linkerd/app/outbound/src/http/tests.rs

```diff
@@ -448,6 +459,78 @@ async fn active_stacks_dont_idle_out() {
         .expect("proxy background task failed");
 }
 
+mod profile_endpoint_override {
+    use super::*;
+
+    async fn test_endpoint_override(profile: profile::Profile) {
+        // Tests that when a profile is resolved with no logical name, but an
+        // endpoint override, the endpoint override is honored.`s`
```

Comment:
If if there is a logical name, as long as `endpoint` is populated we honor that right?